src/DayPicker.js

```diff
@@ -104,14 +104,18 @@ class DayPicker extends Component {
     const baseClass = "DayPicker-NavButton DayPicker-NavButton";
     return (
       <div className="DayPicker-NavBar">
-        <span
+        <button
           key="prev"
           className={ `${baseClass}--prev` }
-          onClick={ ::this.handlePrevMonthClick } />
-        <span
+          onClick={ ::this.handlePrevMonthClick }
+          tabIndex="-1"
+          aria-label={ this.props.localeUtils.getPreviousMonthLabel() } />
```

Comment:
Sorry, I'd ask for another change. We need to pass the `locale` to the function for supporting localization as in the other `format*` functions:
 
```
const { localeUtils, locale } = this.props;
//snip 
aria-label={ localeUtils.getPreviousMonthLabel(locale) }
// snip
aria-label={ localeUtils.getNextMonthLabel(locale) }
```

thanks !